src/base/db.rs

```diff
@@ -81,6 +82,21 @@ pub fn rollback_if_required(conn: &RealmConnection) {
     }
 }
 
+pub fn db_test_conn<F>(run: F)
```

Comment:
Rename this to db_test(). Also check out db_test macro in realm_macros.

Reply:
done